src/Components/FlashCards.js

```diff
@@ -38,27 +40,53 @@ class FlashCards extends Component {
 
   render() {
     const { kanaCounter, isMeaningShown, kanaTable } = this.state;
+    const OuterGrid = styled(Grid)({
```

Comment:
I would put this const before component definition - the way it is right now the OuterGrid component definition is created from scratch on every render